samples/nodejs/quickstart.js

```diff
@@ -33,10 +39,18 @@ async function quickstart() {
     return false;
   });
 
+  const firstPage = await client.assets.list();
```

Comment:
How do we get the Array of items from firstPage or secondPage?

Reply:
@PhilippeBettler like this:
`const myAssets = firstPage.items`